packages/react-reconciler/src/ReactFiberPerformanceTrack.js

```diff
@@ -159,13 +161,64 @@ export function logComponentRender(
   }
 }
 
-export function logSuspenseBoundaryClientRendered(
+export function logComponentErrored(
+  fiber: Fiber,
+  startTime: number,
+  endTime: number,
+  errors: Array<CapturedValue<mixed>>,
+): void {
+  if (supportsUserTiming) {
+    const name = getComponentNameFromFiber(fiber);
+    if (name === null) {
+      // Skip
+      return;
+    }
+    const properties = [];
+    if (__DEV__) {
+      for (let i = 0; i < errors.length; i++) {
+        const capturedValue = errors[i];
+        const error = capturedValue.value;
+        const message =
+          typeof error === 'object' &&
+          error !== null &&
+          typeof error.message === 'string'
+            ? // eslint-disable-next-line react-internal/safe-string-coercion
+              String(error.message)
+            : // eslint-disable-next-line react-internal/safe-string-coercion
+              String(error);
+        properties.push(['Error', message]);
+      }
+    }
+    performance.measure(name, {
+      start: startTime,
+      end: endTime,
+      detail: {
+        devtools: {
+          color: 'error',
+          track: COMPONENTS_TRACK,
+          tooltipText:
+            fiber.tag === SuspenseComponent
+              ? 'Hydration failed'
+              : 'Error boundary caught an error',
+          properties,
+        },
+      },
+    });
+  }
+}
+
+function logComponentEffectErrored(
   fiber: Fiber,
   startTime: number,
   endTime: number,
   errors: Array<CapturedValue<mixed>>,
 ): void {
   if (supportsUserTiming) {
+    const name = getComponentNameFromFiber(fiber);
+    if (name === null) {
+      // Skip
+      return;
```

Comment:
Might be good to include why we can skip this in the comment, seems like it should only be know for the `Throw` case?

Reply:
This is the same the render pass but I don't love this technique. It's just a way for things like internal Fibers to be excluded from DevTools. Like the HostRoot. We use this elsewhere for things like stack frames. It doesn't really actually make sense because like the internal Offscreen component can now sometimes be visible and they should probably have the name Activity at least the way the layering is set up now. We should figure out something more explicit.